acl redis tryLoadMissACLs panic

```
panic: interface conversion: interface {} is string, not []uint8

goroutine 17 [running, locked to thread]:
github.com/juicedata/juicefs/pkg/meta.(*redisMeta).tryLoadMissACLs(0x14000156230, {0x29afe7688?, 0x14000fb2c40}, 0x14000bdf320)
	/Users/tangyoupeng/Work/code/project/juicefs/pkg/meta/redis.go:4779 +0x2b0
github.com/juicedata/juicefs/pkg/meta.(*redisMeta).doMknod.func1(0x14000bdf320)
	/Users/tangyoupeng/Work/code/project/juicefs/pkg/meta/redis.go:1367 +0x5a4
github.com/juicedata/juicefs/pkg/meta.replaceErrno.func1(0x14000d12000?)
	/Users/tangyoupeng/Work/code/project/juicefs/pkg/meta/redis.go:904 +0x24
github.com/redis/go-redis/v9.(*Client).Watch(0x14000fead80?, {0x12a38f318, 0x14000fb2c40}, 0x14000b2a690, {0x14000fbca60, 0x2, 0x2})
	/Users/tangyoupeng/go/pkg/mod/github.com/redis/go-redis/v9@v9.0.2/tx.go:67 +0xb4
github.com/juicedata/juicefs/pkg/meta.(*redisMeta).txn(0x14000156230, {0x29afe7688?, 0x14000fb2c40}, 0x29a74ddb0?, {0x14000fbca60?, 0x2, 0x2})
	/Users/tangyoupeng/Work/code/project/juicefs/pkg/meta/redis.go:937 +0x33c
github.com/juicedata/juicefs/pkg/meta.(*redisMeta).doMknod(0x14000156230, {0x29afe7688?, 0x14000fb2c40}, 0x802, {0x1400005628f, 0x8}, 0x2, 0x1ff, 0x12, 0x0, ...)
	/Users/tangyoupeng/Work/code/project/juicefs/pkg/meta/redis.go:1300 +0x3f0
github.com/juicedata/juicefs/pkg/meta.(*baseMeta).Mknod(0x1400021aa00, {0x29afe7688, 0x14000fb2c40}, 0x802, {0x1400005628f, 0x8}, 0x2, 0xfb?, 0x140?, 0x98057aac?, ...)
	/Users/tangyoupeng/Work/code/project/juicefs/pkg/meta/base.go:1401 +0x210
github.com/juicedata/juicefs/pkg/meta.(*baseMeta).Mkdir(0x1400021aa00, {0x29afe7688?, 0x14000fb2c40?}, 0x802, {0x1400005628f?, 0x299899384?}, 0x8?, 0x0?, 0x0?, 0x1400005f658, ...)
	/Users/tangyoupeng/Work/code/project/juicefs/pkg/meta/base.go:1425 +0x58
github.com/juicedata/juicefs/pkg/fs.(*FileSystem).Mkdir(0x14000215a40, {0x29afe7688?, 0x14000fb2c40}, {0x14000056270, 0x27}, 0x1ff, 0x9afe?)
	/Users/tangyoupeng/Work/code/project/juicefs/pkg/fs/fs.go:441 +0x308
main.jfs_mkdir(0x1400010e000?, 0x0?, 0x16d179301?, 0x2680?, 0x0?)
	/Users/tangyoupeng/Work/code/project/juicefs/sdk/java/libjfs/main.go:783 +0x84
```

--- a/pkg/meta/redis.go
+++ b/pkg/meta/redis.go
@@ -4775,7 +4775,7 @@ func (m *redisMeta) tryLoadMissACLs(ctx Context, tx *redis.Tx) error {
 			var rule *aclAPI.Rule
 			if data != nil {
 				rule = &aclAPI.Rule{}
-				rule.Decode(data.([]byte))
+				rule.Decode([]byte(data.(string)))
 			}
 			// may have empty slot
 			m.aclCache.Put(missIds[i], rule)

